### PR TITLE
Add "minimize all windows" script

### PR DESCRIPTION
**Pull request by namzo**
## Description

Minimize all windows of currently running apps

## Type of change

- [x] New script command

## Screenshot

https://user-images.githubusercontent.com/1392665/154869767-71970f4e-221b-4a6c-b532-70a26448c1ce.mp4

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/system/minimize-all-windows.applescript [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Minimize all windows
```
Comment (dehesa) [inline commands/system/minimize-all-windows.applescript:5]: Raycast uses titlecase for script titles. Therefore: `Minimize All Windows`.

**Review by dehesa — CHANGES_REQUESTED** [review]
Hi @namzo,

Thanks for the contribution. I just left you a small comment.

File: commands/system/minimize-all-windows.applescript (hunk 1 above)
Comment (namzo) [reply commands/system/minimize-all-windows.applescript:5]: Thanks! I'lll fix it now

**Comment by dehesa** [comment]
Great! Thanks again!

**Pull request merged**